app/views/catalog/_form_basic_info.html.haml

```diff
@@ -52,7 +52,20 @@
                         :class                => "selectpicker")
           :javascript
             miqSelectPickerEvent('dialog_id', '#{url}')
-
+    - if @edit[:new][:st_prov_type] == "generic"
+      .form-group
+        %label.col-md-2.control-label
+          = _('Subtype')
+        .col-md-4
+          %p.form-control-static
+            - available_subtypes = @generic_subtypes.invert.to_a.sort_by { |a| a.first.downcase }
+            - options = [["<#{_('No Subtype')}>", nil]] + available_subtypes
```

Comment:
@ZitaNemeckova in most of the forms we use drop down entry as <Choose> so this can be changed to
- options = [["<#{_('Choose')}>", nil]] + available_subtypes